app/presenters/user_presenter.rb

```diff
@@ -46,4 +46,8 @@ def unread_messages_count
   def admin
     user.admin?
   end
+
+  def moderator
```

Comment:
I'd make that a properly syntaxed predicate method (with a "?"). Additionally, you could use the Forwardable module in ruby to use implicit delegation instead of explicit one even though I realize this might be a little overkill here.